sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/api/Azure.ResourceManager.MobileNetwork.netstandard2.0.cs

```diff
@@ -832,6 +837,12 @@ public partial class EncryptedSimUploadList
         public string VendorKeyFingerprint { get { throw null; } }
         public int Version { get { throw null; } }
     }
+    public partial class EventHubConfiguration
+    {
+        public EventHubConfiguration(string id) { }
+        public string Id { get { throw null; } set { } }
```

Comment:
Is this the arm resource identifier of the eventhub we would like to use?
Somehting like `/subscriptions/{}/resourceGroups/{}/providers/Microsoft.EventHub/eventHubs/{name}`?
If so, please add a `arm-id` format to this property.